Dockerfile

```diff
@@ -14,6 +14,5 @@ WORKDIR /opt/app
 
 RUN ./mvnw clean install
 
-COPY ${JAR_FILE} app.jar
```

Comment:
One more nitpick, if the `JAR_FILE` arg is not being used, `docker-compose.yml` should also get rid of the `JAR_FILE` arg for readability

Reply:
fixed

Reply:
fixed